src/v2/guide/components-slots.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Slots
+type: guide
+order: 104
+---
+
+> This page assumes you've already read the [Components Basics](components.html). Read that first if you are new to components.
+
+## Slot Content
+
+Vue implements a content distribution API that's modeled after the current [Web Components spec draft](https://github.com/w3c/webcomponents/blob/gh-pages/proposals/Slots-Proposal.md), using the `<slot>` element to serve as distribution outlets for content.
+
+This allows you to compose components like this:
+
+``` html
+<navigation-link url="/profile">
+  Your Profile
+</navigation-link>
+```
+
+Then in the template for `<navigation-link>`, you might have:
+
+``` html
+<a
+  v-bind:href="url"
+  class="nav-link"
+>
+  <slot></slot>
+</a>
+```
+
+When the component renders, the `<slot>` element will be replaced by "Your Profile". Slots can contain any template code, including HTML:
+
+``` html
+<navigation-link url="/profile">
+  <!-- Add a Font Awesome icon -->
+  <span class="fa fa-user"></span>
+  Your Profile
+</navigation-link>
+```
+
+Or even other components:
+
+``` html
+<navigation-link url="/profile">
+  <!-- Add an icon with a component -->
+  <font-awesome-icon name="user"></font-awesome-icon>
+  Your Profile
+</navigation-link>
+```
+
+If `<navigation-link>` did **not** contain a `<slot>` element, any content passed to it would simply be discarded.
+
+## Compilation Scope
```

Comment:
Reading through, the flow of this seemed out of order to me. I think it would make more sense to address named slots and default slots first, and then this section closer to scoped slots. What do you think?

Reply:
I like that! I just moved that section down. 🙂

Reply:
Thanks!